CardScan/Classes/Torch.swift

```diff
@@ -9,6 +9,11 @@ struct Torch {
     let device: AVCaptureDevice?
     var state: State
     var lastStateChange: Date
+    var level: Float {
+        didSet {
```

Comment:
I don't think that we've used property observers anywhere else, I don't think we should introduce them for this change

Reply:
well the reason I used the observer is because once the property is set, it wont be set into torchMode until the `setTorchMode` function is called. so without this, the intial torch level will be 1.0, then when set to some value, it will only be changed after a toggle.

Reply:
wouldn't you always have the level when you turn the torch on for the first time? I'm probably just not following something.

Reply:
hmmm... you're right. let me update.